modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -695,8 +706,9 @@ public void registerObjectRelationshipsRelatedInfoCollectionProviders(
 	@Indexable(type = IndexableType.REINDEX)
 	@Override
 	public ObjectRelationship updateObjectRelationship(
-			long objectRelationshipId, long parameterObjectFieldId,
-			String deletionType, Map<Locale, String> labelMap)
+			String externalReferenceCode, long objectRelationshipId,
```

Comment:
As we discuss, add externalReferenceCode property in  _updateObjectRelationship()